### ISSUE TITLE
First Use Hangs

### ISSUE THREAD

**Issue by resistorsoftware** [Microsoft/vscode-postgresql#39]
- PGSQL Extension Version: 0.2.0
- VSCode Version: 1.3.3.0
- OS Version: MacOS 10.14.4

Steps to Reproduce:

1. Shift+CTRL P
2. Select New Query

Result is a message that the extension is unavailable till it loads. Repeat, extension then works.

**Comment by akamath46297** [comment]
@resistorsoftware , when you install the extension for the first time, it has to download and start the tools service which might take few seconds based on your network bandwidth. You could see this in the output window similar to what I have shown below. You will not be able to use the extension until this is done. This is exactly what the message states.  Let me know if I am interpreting your issue differently.

![image](https://user-images.githubusercontent.com/30480046/56069041-00876080-5d36-11e9-8771-c054489b10ed.png)

**Comment by resistorsoftware** [comment]
Interestingly, since the last time I used the extension, doing nothing much at all except cruising some open windows, I decided to try the extension again. But not before noticing that the settings.json for connections was a lot more detailed. Ok! So that tells me something in the extension kicked in and wrote details for itself with more pizazz. 

So I tried a query and it worked. And then I added a second connection, and noticed that with no effort, the list of connections was visible, I could choose one, and direct my query at that connection. 

Interesting then, that this extension went from broken to updated, to still kinda broken, to working as expected, all within a day, and without me doing anything.
